src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/DiskRPTests.ps1

```diff
@@ -137,17 +139,19 @@ function Test-Snapshot
         Assert-AreEqual $false $snapshot.EncryptionSettings.Enabled;
 
         # Grant access test
-        Grant-AzureRmSnapshotAccess -ResourceGroupName $rgname -SnapshotName $snapshotname -Access $access -DurationInSecond 5;
-        Revoke-AzureRmSnapshotAccess -ResourceGroupName $rgname -SnapshotName $snapshotname;
+        $job = Grant-AzureRmSnapshotAccess -ResourceGroupName $rgname -SnapshotName $snapshotname -Access $access -DurationInSecond 5 -AsJob;
+        $job | Wait-Job;
+        Revoke-AzureRmSnapshotAccess -ResourceGroupName $rgname -SnapshotName $snapshotname -AsJob;
+        $job | Wait-Job;
```

Comment:
Can you add asserts for Grant-AzureRmSnapshotAccess and Revoke-AzureRmSnapshotAccess to ensure they have succeeded?